lib/npm.js

```diff
@@ -92,7 +92,7 @@
   var commandCache = {}
   var aliasNames = Object.keys(aliases)
 
-  var littleGuys = [ 'isntall', 'verison' ]
+  var littleGuys = [ 'install', 'verison' ]
```

Comment:
i'm reasonably sure these typos (version, as well) are intentional, and are to make it so that `npm isntall` and `npm verison` still work even though the user made a typo.

(ie, "these little guys? i wouldn't worry about these little guys")